src/components/core/components/Link/Link.tsx

```diff
@@ -20,8 +20,8 @@ type Props = BaseLinkProps &
     | RouterLinkProps
   );
 
-const StyledBaseLink = styled(({ fullWidth, noWrap, ...rest }) => (
-  <BaseLink {...rest} />
+const StyledBaseLink = styled(({ fullWidth, noWrap,to, ...rest }) => (
+  <BaseLink to={to ?? "/"} {...rest} />
```

Comment:
the parameter "to" will not have default value "/". the correct default value is undefined. please remove this change completly

Reply:
fixed